firmware/src/src/defines.h

```diff
@@ -17,6 +17,7 @@
 #define CALCULATE_PERIOD 6666   // (us) 150hz IMU calculations
 #define PWM_FREQUENCY 50        // (ms) PWM Period
 #define UIRESPONSIVE_TIME 10000 // (ms) 10Seconds without an ack data will stop;
+#define RESET_PULSE_PERIOD 500000 // (us) pulse duration for the reset signal to TX
```

Comment:
Is this a lingering edit. I agree it should be in trackersettings like you have it.